ui/tests/acceptance/secrets/backend/database/secret-test.js

```diff
@@ -229,12 +229,6 @@ const connectionTests = [
       assert
         .dom('[data-test-input="root_rotation_statements"]')
         .exists(`Root rotation statements exists for ${name}`);
-      assert
-        .dom('[data-test-alert-banner="alert"]')
-        .hasTextContaining(
-          `Warning Please ensure that your Oracle plugin has the default name of vault-plugin-database-oracle. Custom naming is not supported in the UI at this time. If the plugin is already named vault-plugin-database-oracle, disregard this warning.`,
```

Comment:
This test is referencing a different banner, right? Is this banner also going to be removed, or should this test still be here

Reply:
Ah, yeah, I added coverage for Arnav's banner when I noticed it didn't have any. I'll add that back in. Good catch.